seatunnel-common/src/test/java/org/apache/seatunnel/common/utils/JsonUtilsTest.java

```diff
@@ -0,0 +1,218 @@
+        Assert.assertEquals("192.168.xx.xx", maps.get(0).get("mysql address"));
+        Assert.assertEquals("3306", maps.get(0).get("port"));
+        Assert.assertEquals("80", maps.get(0).get("no index of number"));
+        Assert.assertEquals("190", maps.get(0).get("database client connections"));
+    }
+
+    public String list2String() {
+
+        LinkedHashMap<String, String> map1 = new LinkedHashMap<>();
+        map1.put("mysql service name", "mysql200");
+        map1.put("mysql address", "192.168.xx.xx");
+        map1.put("port", "3306");
+        map1.put("no index of number", "80");
+        map1.put("database client connections", "190");
+
+        List<LinkedHashMap<String, String>> maps = new ArrayList<>();
+        maps.add(0, map1);
+        String resultJson = JsonUtils.toJsonString(maps);
+        return resultJson;
+    }
+
+    @Test
+    public void testParseObject() {
+        Assert.assertNull(JsonUtils.parseObject(""));
+        Assert.assertNull(JsonUtils.parseObject("foo", String.class));
+    }
+
+    @Test
+    public void testNodeString() {
+        Assert.assertEquals("", JsonUtils.getNodeString("", "key"));
+        Assert.assertEquals("", JsonUtils.getNodeString("abc", "key"));
+        Assert.assertEquals("", JsonUtils.getNodeString("{\"bar\":\"foo\"}", "key"));
+        Assert.assertEquals("foo", JsonUtils.getNodeString("{\"bar\":\"foo\"}", "bar"));
+        Assert.assertEquals("[1,2,3]", JsonUtils.getNodeString("{\"bar\": [1,2,3]}", "bar"));
+        Assert.assertEquals("{\"1\":\"2\",\"2\":3}", JsonUtils.getNodeString("{\"bar\": {\"1\":\"2\",\"2\":3}}", "bar"));
+    }
+
+    @Test
+    public void testJsonByteArray() {
+        String str = "foo";
+        byte[] serializeByte = JsonUtils.toJsonByteArray(str);
+        String deserialize = JsonUtils.parseObject(serializeByte, String.class);
+        Assert.assertEquals(str, deserialize);
+        str = null;
+        serializeByte = JsonUtils.toJsonByteArray(str);
+        deserialize = JsonUtils.parseObject(serializeByte, String.class);
+        Assert.assertNull(deserialize);
+    }
+
+    @Test
+    public void testToList() {
+        Assert.assertEquals(new ArrayList(),
+                JsonUtils.toList("A1B2C3", null));
+        Assert.assertEquals(new ArrayList(),
+                JsonUtils.toList("", null));
+    }
+
+    @Test
+    public void testCheckJsonValid() {
+        Assert.assertTrue(JsonUtils.checkJsonValid("3"));
+        Assert.assertFalse(JsonUtils.checkJsonValid(""));
+    }
+
+    @Test
+    public void testFindValue() {
+        Assert.assertNull(JsonUtils.findValue(
+                new ArrayNode(new JsonNodeFactory(true)), null));
+    }
+
+    @Test
+    public void testToMap() {
+        Map<String, String> map = new HashMap<>();
+        map.put("foo", "bar");
+
+        Assert.assertTrue(map.equals(JsonUtils.toMap(
+                "{\n" + "\"foo\": \"bar\"\n" + "}")));
```

Comment:
This one can be simplified, as follows:

```suggestion
        Assert.assertEquals(map, JsonUtils.toMap(
                "{\n" + "\"foo\": \"bar\"\n" + "}"));
```